lib/nacre/channel.rb

```diff
@@ -27,20 +27,12 @@ def warehouse_ids=(hash)
 
     private
 
-    def self.request_options
-      ''
-    end
-
     def self.service_name
-      'channel'
+      'product'
     end
 
-    def self.service_url
-      configuration.resource_url + '/' + 'product-service'
-    end
-
-    def self.url
-      service_url + '/' + service_name
+    def self.request_options
+      ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.